### PR TITLE
[DI] Automatically applying a tag to all services implementing an int…

### PR DESCRIPTION
**Pull request by danut007ro**
…erface

### CODE REVIEW

**Comment by GuilhemN** [comment]
What about adding this as a tip instead, and use the `Kernel::build()` in the example as it's becoming more common?

```rst
.. tip::

    To apply a tag to all your autoconfigured services extending a class or an
    interface, call ``ContainerBuilder::registerForAutoconfiguration()`` in an
    :doc:`extension </bundles/extension>` or from your kernel::

        // app/AppKernel.php
        class AppKernel extends Kernel
        {
            // ...

            protected function build(ContainerBuilder $container)
            {
                $container->registerForAutoconfiguration(CustomInterface::class)
                    ->addTag('app.custom_tag')
                ;
            }
        }
```

**Comment by danut007ro** [comment]
Even better. Updated.

**Review by GuilhemN — APPROVED** [review]
:+1:, thanks for your first contribution!

File: service_container/tags.rst [hunk 1]
```diff
@@ -70,6 +70,25 @@ then some tags are automatically applied for you. That's true for the ``twig.ext
 tag: the container sees that your class extends ``Twig_Extension`` (or more accurately,
 that it implements ``Twig_ExtensionInterface``) and adds the tag for you.
 
+.. tip::
+
+    To apply a tag to all your autoconfigured services extending a class or an
+    interface, call ``ContainerBuilder::registerForAutoconfiguration()`` in an
```
Comment (xabbuh) [inline service_container/tags.rst]: We can even use the `method` role here to create a link to the API docs:

```rst
[...] :method:`Symfony\\Component\\DependencyInjection\\ContainerBuilder::registerForAutoconfiguration` [...]
```

**Review by xabbuh — APPROVED** [review]

**Comment by danut007ro** [comment]
Updated, thanks @xabbuh

**Comment by weaverryan** [comment]
Awesome! Thank you Daniel!

**Pull request merged**